rest/bulk_api.go

```diff
@@ -188,6 +188,12 @@ func (h *handler) handleAllDocs() error {
 			if err != nil {
 				return false, err
 			}
+			if row.Doc != nil {
+				base.Audit(h.ctx(), base.AuditIDDocumentRead, base.AuditFields{
+					base.AuditFieldDocID:      row.ID,
+					base.AuditFieldDocVersion: row.Value.Rev,
+				})
+			}
```

Comment:
The other non-document information sent with _all_docs should be a Document Metadata Read, I think?  (channels, revid, etc)

Reply:
I don't know if we should log channels when getting the DocumentMetadataRead?

I don't know what it tells you because channels aren't getting modified. I'm not opposed to logging this information.